src/client/src/components/Quiz/Quiz.jsx

```diff
@@ -0,0 +1,12 @@
+import "./Quiz.css";
+import BasicQuestion from "./BasicQuestion/BasicQuestion";
```

Comment:
About naming of css files — we start from a capital letter?

Reply:
I did with capital because I taught its the convention but maybe I'm wrong...?
tell me if its need to be changed